### DISCUSSION TITLE
navigator.mediaDevices

### DISCUSSION THREAD

**Discussion by joshmkennedy** [progrium/topframe#13] [Q&A]
Is it possible to show the feed of the camera on a video element. It doesn't seem to work using navigator.mediaDevices

**Comment by progrium** [comment]
no webkit has that turned off and dont seem to expose an api to change
that. i was planning to try and stream in video via datastream. if theres a
cli tool for producing a stream of your webcam it could be piped in via
script

On Wed, Sep 1, 2021 at 11:53 AM Josh Kennedy ***@***.***>
wrote:

> Is it possible to show the feed of the camera on a video element. It
> doesn't seem to work using navigator.mediaDevices
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/progrium/topframe/discussions/13>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAAFB66CJYLUZMJ5Q2WTOTT7ZK7HANCNFSM5DG5D3EA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
-- 
Jeff Lindsay
http://progrium.com
